lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -139,6 +148,33 @@ def get_request_id(headers, env)
           headers ||= {}
           headers['X-Request-Id'] || headers['X-Request-ID'] || env['HTTP_X_REQUEST_ID']
         end
+
+        private
+
+        REQUEST_SPAN_DEPRECATION_WARNING = %(
+          Use of :datadog_rack_request_span in the Rack env is DEPRECATED.
```

Comment:
I would say: "`:datadog_rack_request_span` is an internal symbol in the Rack env and is DEPRECATED. If you need (etc...)"